source/shared/net/guides/run-node.rst

```diff
@@ -47,15 +47,15 @@ To upgrade do the following steps after downloading the new version.
    ``--no-block-state-import`` to avoid downloading data you already have.
 
 .. note::
-   If your node does not show on the network dashboard after upgrading to 1.1.3,
+   If your node does not show on the network dashboard after upgrading to 3.0.0,
    it could be that the node was started by the old version of the runner.
    The old version of the software does not configure the docker container correctly, and this
    misconfiguration can persist even if you then use the new version.
    To fix this:
 
    #. Ensure that you have downloaded and extracted the :ref:`latest version of the software<concordium-docker-package-download>`.
 
-   #. Check the version by running ``concordium-node --help``. It should show 1.1.3.
+   #. Check the version by running ``concordium-node --help``. It should show 3.0.0.
```

Comment:
We'll need to double check this. The currently built version will still output 1.1.3, but I've asked that this be changed.

I'll get back to this once that is resolved.

Reply:
It will be like you wrote, so I'll close this conversation.